checkbox-ng/checkbox_ng/launcher/master.py

```diff
@@ -312,6 +312,8 @@ def new_session(self):
 
         try:
             tps = self.sa.start_session(configuration)
+            if self.sa.sideloaded_providers:
+                _logger.warning("Using sideloaded providers")
```

Comment:
To ensure that someone doesn't assume it's the controller that has side-loaded stuff, I'd add:
"Agent is using..."